README.md

```diff
@@ -1,33 +1,40 @@
 # Welcome
 
-This repository contains the source code for:
+This repository contains the source code & documentation for:
 
 * [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
 * The Windows console host (`conhost.exe`)
 * Components shared between the two projects
 * [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
 * [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
-  
+
 Other related repositories include:
 
 * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
 
 ## Installing & running Windows Terminal
 
-> 👉 Note: To run the Windows Terminal you'll need to be running Windows 10 1903 (build 18362) or later
+### Operating System Support
+
+Windows Terminal can be installed on:
+
+* Windows 10 1903 (build 18362) or later
+* [Windows Server 1903](https://docs.microsoft.com/en-us/windows-server/get-started-19/whats-new-in-windows-server-1903) or later (install manually - see "From this repository" below)
```

Comment:
Is this correct? Windows Server semi-annual channel releases like 1903 cannot be installed with the full desktop GUI, only as Server Core - which is basically only enough to run Notepad, Task Manager, the original console window, and Server Manager. Hence, I'd be surprised if they run any graphical applications of even moderate complexity, including classic desktop apps that use UWP APIs like Terminal.

Reply:
Good catch. Thanks. fixing